src/pybind/mgr/dashboard/.pylintrc

```diff
@@ -393,7 +393,7 @@ ignored-classes=optparse.Values,thread._local,_thread._local
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=cherrypy,distutils
+ignored-modules=cherrypy,distutils,rados,rbd,cephfs
```

Comment:
Adding comments doesn't make any harm, but I'm not sure if that's the reason I'd add. Rather than a matter of build order, I'd say it's a matter of relevance (should we necessarily build 3rd parties to quality-check dashboard code?).